Add touch screen support

burned42 opened the pull request:
I found this repo due to the hacktoberfest label and thought I'd tackle this issue :)

Closes https://github.com/SiddharthaBhattacharjee/TimePassGames/issues/12

![grafik](https://user-images.githubusercontent.com/16883833/194712665-ae5a8604-84a4-4e76-a9bf-ba6684f0d51e.png)
I decided to place the touch controls at the middle of the screen as this works good in either orientation, portrait or horizontal. They won't be shown if the device doesn't support touch.
Feel free to suggest a different placement/size if you have other a different opinion.

Since this alone is not very useful as the main page also says that touch isn't supported, I also removed that message and checked if the remaining games are playable on my smartphone and noticed that most of them already did and for Mix-match it was also easy to add touch support as it was only about starting the game, the rest already worked.

I then also adjusted the main page to use flexbox for aligning the games so the list is usable on mobile, too, as the list was overflowing to the left and right on my mobile and I couldn't see all games.

So at the end I can now open the main page on my smartphone and I was able to play all the games with touch input.

I enabled github pages on my branch, so you can see the changes at work here: https://burned42.github.io/TimePassGames/

SiddharthaBhattacharjee commented:
Thanks a lot for your contribution, We have checked it out and we are very impressed by your work, there is a small bug in the snake game where the food occasionally spawns outside the screen, but other than that everything is working great. Also thanks for working on the home page in addition to the snake game. I am merging this PR and We will address that small bug in near future.

burned42 commented:
@SiddharthaBhattacharjee indeed, I couldn't imagine that I introduced that bug, but I was able to reproduce it and fixed it in https://github.com/SiddharthaBhattacharjee/TimePassGames/pull/22
